content/2017/02/17/color-theory-for-coders.rst

```diff
@@ -0,0 +1,37 @@
+public: yes
+author: sondra
+tags: [Tips, Color, Design, Recommendations]
+image:
+  - src: '2017/color-theory/decision-tree.jpg'
+summary: |
+  If you write code and you want to delve into design, Natalya Shelburne's Practical Color Theory for Coders will get you up and running with a beautiful, cohesive, and accessible color palette using Sass color functions and teach you why her palettes work so well all at the same time.
+
+
+Birds Recommend: Color Theory for Coders
+=======================================
+
+So you write code, but design intrigues you and you'd like to delve deeper. But where to start? The sheer number of tools and tutorial is overwhelming. My article on the best design tools and tutorials will go live next week, but while you're waiting, checkout `Natalya Shelburne's`_ `Practical Color Theory for People Who Code`_. Continue reading for a quick summary or **head
+over to Natalya's article right away!**
+
+.. _Natalya Shelburne's: https://twitter.com/natalyathree
+.. _Practical Color Theory for People Who Code: http://tallys.github.io/color-theory/
+
+`Go to Color Theory >`_
```

Comment:
These don't seem like headers to me. Maybe use buttons instead? See my CSS Conf recap post for how I did that with raw HTML.